src/Servers/IIS/IIS/src/IISServerOptions.cs

```diff
@@ -2,12 +2,14 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Runtime.Versioning;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
 using Microsoft.AspNetCore.Server.IIS;
 
 namespace Microsoft.AspNetCore.Builder
 {
+    [SupportedOSPlatform("windows")]
```

Comment:
I thought you were going to remove this one? Trying to use it doesn't break anything. All the warning does is require suppressions for cross deployed code (e.g. developed on mac, published to windows).

Reply:
JFYI it shouldn't warn because of developing in another platform, it would warn if the app not targeting windows or multi-targeted, (MsBuild should provide the target platform for the analyzer)